### PR TITLE
Update documentation to include route-serializer info

### PR DESCRIPTION
**Pull request by trentmwillis**
As requested in emberjs/ember.js#13016 and discussed in the RFC: [Route Serializers](https://github.com/emberjs/rfcs/blob/master/text/0120-route-serializers.md). Let me know if anything else is needed as I haven't contributed to the guides before.

### CODE REVIEW

**Comment by rwjblue** [comment]
Awesome, thank you @trentmwillis.  We will likely hold this PR and merge closer to the release version of the feature, but I defer to the @emberjs/guides-managers on that...

**Comment by locks** [comment]
I confirm, it will get merged when we do the review period freeze so we can get updates to the current content in as much as possible. 
Appreciate the work! :)

File: source/localizable/routing/defining-your-routes.md [hunk 1]
```diff
@@ -183,6 +183,26 @@ If the user navigates to `/post/5`, the route will then have the `post_id` of
...
+
+Router.map(function() {
+  this.route('post', {
+    path: '/post/:post_id',
+    serialize: serializePost
+  });
+});
+```
+
+If no `serialize` function is defined, then a straight property lookup on the model will be used for each segment instead.
```
Comment (locks) [inline source/localizable/routing/defining-your-routes.md]: Can you give an example? Or maybe inverse it, first explain how the default works, and then explain how you can customize. I believe you can check the `model()` hook documentation for a similar flow.

Comment (trentmwillis) [inline source/localizable/routing/defining-your-routes.md]: Makes sense. Will update.

**Comment by trentmwillis** [comment]
@locks updated. Let me know what you think.

**Pull request closed without merging**